resources/install.rb

```diff
@@ -1,39 +1,74 @@
+          default: '/usr'
+
+property :sensitive, [true, false],
+          default: true
 
 # Package
-property :package_name, String, default: 'haproxy'
+property :package_name, String,
+          default: 'haproxy'
+
 property :package_version, [String, nil]
-property :enable_ius_repo, [true, false], default: false
-property :enable_epel_repo, [true, false], default: true
+
+property :enable_ius_repo, [true, false],
+          default: false
+
+property :enable_epel_repo, [true, false],
+          default: true
 
 # Source
-property :source_version,     String, default: '2.2.4'
-property :source_url,         String, default: lazy { "https://www.haproxy.org/download/#{source_version.to_f}/src/haproxy-#{source_version}.tar.gz" }
-property :source_checksum,    [String, nil], default: '87a4d9d4ff8dc3094cb61bbed4a8eed2c40b5ac47b9604daebaf036d7b541be2'
-property :source_target_cpu,  [String, nil], default: lazy { node['kernel']['machine'] }
-property :source_target_arch, [String, nil]
-property :source_target_os,   String, default: lazy { target_os(source_version) }
-property :use_libcrypt,       String, equal_to: %w(0 1), default: '1'
-property :use_pcre,           String, equal_to: %w(0 1), default: '1'
-property :use_openssl,        String, equal_to: %w(0 1), default: '1'
-property :use_zlib,           String, equal_to: %w(0 1), default: '1'
-property :use_linux_tproxy,   String, equal_to: %w(0 1), default: '1'
-property :use_linux_splice,   String, equal_to: %w(0 1), default: '1'
-property :use_lua,            String, equal_to: %w(0 1), default: '0'
-property :lua_lib,            [String, nil]
-property :lua_inc,            [String, nil]
-property :use_systemd,        String, equal_to: %w(0 1), default: lazy { source_version.to_f >= 1.8 ? '1' : '0' }
+property :source_version, String,
+          default: '2.2.4'
+
+property :source_url, String,
+          default: lazy { "https://www.haproxy.org/download/#{source_version.to_f}/src/haproxy-#{source_version}.tar.gz" }
+
+property :source_checksum, String,
+          default: '87a4d9d4ff8dc3094cb61bbed4a8eed2c40b5ac47b9604daebaf036d7b541be2'
+
+property :source_target_cpu, String,
+          default: lazy { node['kernel']['machine'] }
+
+property :source_target_arch, String
+
+property :source_target_os, String,
+          default: lazy { target_os(source_version) }
+
+property :use_libcrypt, [true, false],
+          default: true
+
+property :use_pcre, [true, false],
+          default: true
+
+property :use_openssl, [true, false],
+          default: true
+
+property :use_zlib, [true, false],
+          default: true
+
+property :use_linux_tproxy, [true, false],
+          default: true
+
+property :use_linux_splice, [true, false],
+          default: true
+
+property :use_lua, [true, false],
+          default: false
+
+property :lua_lib, String
+
+property :lua_inc, String
+
+property :use_systemd, [true, false],
+          default: lazy { source_version.to_f >= 1.8 }
```

Comment:
This lazy can go away now probably

Reply:
The tables turn 😂. Although i'll just double check but I'm 99% other properties are another edge case where that needs to stay.

Reply:
Yes unfortunately

```
NameError
---------
undefined local variable or method `source_version' for #<Class:0x00000000036b8b60>
```